PencilCase/PencilCase/scripts/Pages/Wholesale.ts

```diff
@@ -0,0 +1,164 @@
+﻿import { PageBase } from './PageBase';
```

Comment:
Copied from ImportProduct.ts, updated as Wholesale page. 
Retail page is similar.
Will do code refactoring to extract a base class when 3 pages are done.